src/views/DocumentationView.js

```diff
@@ -34,7 +34,7 @@ class Documentation extends React.Component {
         <DocumentComponent
           key={title}
           title= {title}
-          component= <CurrComp />
+          component= <CurrComp /> 
```

Comment:
This syntax is wrong 
```suggestion
          component={<CurrComp />}
```